src/main/play-doc/developer/DevIntro.md

```diff
@@ -1,9 +0,0 @@
-# Developer introduction
```

Comment:
I still think that this introduction is somewhat important. It sets the scene. I felt that it was particularly important to emphasise that we're big on avoiding vendor-lockin.